.circleci/config.yml

```diff
@@ -50,9 +50,6 @@ workflows:
                 - gemfiles/rails6.1_graphql1.12.gemfile
                 - gemfiles/rails6.1_graphql1.13.gemfile
                 - gemfiles/rails6.1_graphql2.0.gemfile
-                - gemfiles/rails7.0_graphql1.11.gemfile
-                - gemfiles/rails7.0_graphql1.12.gemfile
-                - gemfiles/rails7.0_graphql1.13.gemfile
```

Comment:
Matrix limits, we'll continue to test these GQL versions only with the oldest Rails version that is tested 6.1